google/ggrc-core
# GGRC-4291 Rearrange comment fields

# Issue description

'Comments' and 'Last Comment' columns should be put after 'Verifiers' and before GCAs in exported assessment and after GCAs on the export page. 

# Steps to test the changes

1) Go to the export page; 'Comments' and 'Last Comment' fields should appear before GCAs;
2) Export assessment csv template; 'Comments' and 'Last Comment' fields should be put after 'Verifiers' and before GCAs;
3) Have created assessment. Export assessment. 'Comments' and 'Last Comment' fields should be put after 'Verifiers' and before GCAs;

# Solution description

For the case 1: required fields are rearranged in the `ATTRIBUTE_ORDER` fields.
For the cases 2 and 3: `SPECIFIC_ORDER` setting has been created to provide additional custom order.

# Sanity checklist




# PR Review checklist
